src/components/SelectBox.js

```diff
@@ -43,16 +44,32 @@ class SelectBox extends React.Component {
       this.props.onChange(id)
     }
   }
+  getValidationState () {
+    const { validationState } = this.props
+    switch (validationState) {
+      case 'error':
+        return style['selectBox-has-error']
+      case 'warning':
+        return style['selectBox-has-warning']
+      case 'success':
+        return style['selectBox-has-success']
+      case 'blinking':
+        return style['selectBox-should-blink']
```

Comment:
I think we don't need this 'blinking' case anymore. Also I'm not sure if we need other than error case here.

Reply:
I partially agree,
I think the blinking option is not necessary any more, but the others maybe will be necessary in the future,
it's just  component's functionality,
better keep it to create consistency with PF3's input components.

Reply:
> ... but the others maybe will be necessary in the future, ...

I understand that you want to support consistency but I am not sure... I would better avoid adding any unnecessary code, a.k.a any code which is not useful/necessary right now. I think we are not sure if this will really be used in the future. And if yes, we still can add later what we will need.